apps/mosaico/src/Main.purs

```diff
@@ -292,20 +295,32 @@ frontpage env { guards: { credentials } } = do
     <$> maybe (pure Nothing) (parallel <<< getUser) credentials
     <*> parallel (getFrontpage mosaicoPaper "Startsidan")
     <*> parallel (Lettera.getMostRead 0 10 Nothing mosaicoPaper true)
-  mosaico <- liftEffect MosaicoServer.app
-  let frontpage' = Frontpage.render
-                     { content: Just articles
-                     , onArticleClick: const mempty
-                     , onTagClick: const mempty
-                     }
-  let mosaicoString =
+  let mosaico = MosaicoServer.app
+      mostReadArticlesProps = { mostReadArticles, onClickHandler: const $ pure unit }
+      frontpage :: ArticleFeed -> JSX
+      frontpage (ArticleList list) = 
```

Comment:
Remove trailing white space. I'm not going to flag all of them this time, there are 16 lines with it, this included. `git diff origin/master origin/mosaico/integrate-andra-laser` marks them with red, at least.